src/policy/concrete.rs

```diff
@@ -19,6 +19,7 @@ use bitcoin::hashes::hex::FromHex;
 use bitcoin::hashes::{hash160, ripemd160, sha256, sha256d};
 use std::{error, fmt, str};
 
+use super::ENTAILMENT_RECURSION_DEPTH;
```

Comment:
This should have a different name, it is a limit on the number of constraints rather than the recursion depth